include/geode/model/helpers/convert_model_meshes.h

```diff
@@ -28,21 +28,33 @@
 namespace geode
 {
     class BRep;
+    class BRepBuilder;
     class Section;
+    class SectionBuilder;
 } // namespace geode
 
 namespace geode
 {
     void opengeode_model_api convert_surface_meshes_into_triangulated_surfaces(
         BRep& brep );
+    void opengeode_model_api convert_surface_meshes_into_triangulated_surfaces(
```

Comment:
Should we deprecate the other one? 
I think both should be kept.

Reply:
I agree